mrc-ide/packit
# Mrc 5661- FE Run page - packet groups + params

Two* fields are added to the run page, as shown in the screenshot. The first is selecting the packet group name. After the user has selected, parameters can be chosen if there are parameters available for the packet group. 

For users to enter params it will be a text box and users will write as strings. And the value gets converted to number, string, boolean depending on what is typed.

Testing: 
1. click on a packet group without params. the params would not show.
2. click on packet group with params (parameters, test1) and parameters should show. play around with typing in different values and clicking run. if you look at console this is what will be sent to server when the submit functionality is done in next pr.

![image](https://github.com/user-attachments/assets/50910b74-406e-4afc-b9e3-ed0893d99efd)
